storage/innobase/row/row0merge.cc

```diff
@@ -1707,7 +1707,8 @@ row_merge_read_clustered_index(
 	ut_stage_alter_t*	stage,
 	double 			pct_cost,
 	row_merge_block_t*	crypt_block,
-	struct TABLE*		eval_table)
+	struct TABLE*		eval_table,
+	bool			drop_historical)
```

Comment:
@vuvova Could the `drop_historical` parameter be removed, and could we handle the versioning columns as hidden columns, and internally generate DROP_COLUMN operation?